internal/services/apimanagement/api_management_api_resource.go

```diff
@@ -393,6 +393,9 @@ func resourceApiManagementApiCreateUpdate(d *pluginsdk.ResourceData, meta interf
 	// If import is used, we need to send properties to Azure API in two operations.
 	// First we execute import and then updated the other props.
 	if vs, hasImport := d.GetOk("import"); hasImport {
+		// as `import` field won't return by API, we need to use `Partial` to avoid set to it state
+		// or the sequential apply will report no changes: https://github.com/hashicorp/terraform-provider-azurerm/issues/24379#issuecomment-1899251199
+		d.Partial(true)
```

Comment:
I feel this is symptomatic of the combined `CreateUpdate`, can we split this before continuing with this custom poller to make this less difficult to maintain?